src/client/api/events/sub_messages_metadata.c

```diff
@@ -12,10 +12,10 @@ int event_subscribe_msg_metadata(event_client_handle_t client, int *mid, char co
     printf("[%s:%d]: Message Id length is invalid\n", __func__, __LINE__);
     return -1;
   }
-  // Buffer to store topic string : messages/{messageid}/metadata
-  char topic_buff[BIN_TO_HEX_BYTES(IOTA_MESSAGE_ID_BYTES) + 21] = {0};
+  // Buffer to store topic string : messages-metadata/0x{messageid}
+  char topic_buff[BIN_TO_HEX_BYTES(IOTA_MESSAGE_ID_BYTES) + 20] = {0};
```

Comment:
the length of "messages-metadata/0x" string is 21.

Reply:
Sorry, topic had a change. It is "message-metadata/0x", 's' was removed from message's' ie why length is 20. I have updated the comment.